src/NimBLEAdvertisedDevice.cpp

```diff
@@ -581,11 +583,11 @@ bool NimBLEAdvertisedDevice::haveTXPower() {
 } // haveTXPower
 
 
-uint8_t NimBLEAdvertisedDevice::findAdvField(uint8_t type, uint8_t index, uint8_t *data_loc) {
+uint8_t NimBLEAdvertisedDevice::findAdvField(uint8_t type, uint8_t index, size_t * data_loc) {
```

Comment:
Hi @h2zero I just happen to stumble across this change and am a bit confused. Shouldn't the parameters rather be `uint8_t type, size_t index, uin8_t* data_loc`? A pointer of type `size_t` seems unusual to me. But maybe I am missing something?

Reply:
Yes it looks odd but the data_loc parameter passed to here is being used by the calling function to access the vector[data_loc] which could be > 255 bytes now. It could have also been an int but size_t seems more appropriate for vector indexing.